ideascube/monitoring/forms.py

```diff
@@ -73,16 +74,17 @@ class Meta:
 
 class InventorySpecimenForm(forms.ModelForm):
 
-    specimen = forms.CharField(widget=forms.TextInput(attrs={
-                               'placeholder': _('Enter a bar code')}))
+    specimen = forms.CharField(label=_('specimen'),
+                               widget=forms.TextInput(attrs={
+                                'placeholder': _('Enter a barcode')}))
```

Comment:
my linter does not complain so neither do I ;)
If you have a better suggestion, please be explicit :)